Networking/Networking/Model/Product/AIProduct.swift

```diff
@@ -125,7 +125,7 @@ public extension Product {
          tags: [ProductTag]) {
         self.init(siteID: siteID,
                   productID: 0,
-                  name: aiProduct.name,
+                  name: aiProduct.names.first ?? "",
```

Comment:
Not a blocker, and we can add it in a future PR:

I think we should add another helper method to init a `Product` with a custom selected name, description and shortDescription. We can use the new custom `init` method to pass the user selected/typed name, description and shortDescription.